src/app/games/diaballik/tests/diaballik.component.spec.ts

```diff
@@ -54,6 +66,15 @@ describe('DiaballikComponent', () => {
         // Then it should succeed
         await testUtils.expectMoveSuccess('#click_4_6', move);
     }));
+    it('should deselect current piece when clicking on it a second time', fakeAsync(async() => {
+        // Given a state where a piece has been selected
+        await testUtils.expectClickSuccess('#click_0_6');
+        // When clicking on it a second time
+        testUtils.expectElementToHaveClass('#piece_0_6', 'selected-stroke');
+        await testUtils.expectClickSuccess('#click_0_6');
+        // Then it should not be selected anymore
+        testUtils.expectElementNotToHaveClass('#piece_0_6', 'selected-stroke');
```

Comment:
j'aime le pattern // Given \n // When \n // Then, ça peut être bien d'aérer !
En plus du fait que le "séparer les méthodes" et "séparer les tests" par un /n est  pourrait aussi être cool içi !
(et tester ce nouveau bonhomme de reviewage dont t'as parlé aussi !)

Reply:
ça risque pas de faire TROP aéré avec des \n partout entre it et entre commentaires ? Un test de 4 lignes de code fera donc 4 lignes (code) + 3 lignes (commentaires) + 2 lignes (\n avant when et avant then) + 2 lignes (avant et après le it) = 11 lignes.
Ça arrive quand même rarement les conflits où les it séparés auraient aidés ? J'en ai jamais vu en tout cas.

Reply:
Bah le test c'est du GIVEN/WHEN/THEN, les séparer ce serait pas pire pour la lecture, vu que c'est des concepts TRES séparés (j'ai vu des projets ou les given when et then son des méthodes ayant le nom de ce que nous on met comment // GWT Commentaire :D !
Donc estime toi chanceux et mets ces espaces haha :D !

Quand aux conflits évidemment, ils n'arrivent que quand on s'y attends pas :D

Reply:
c'est aéré désormais !